Avoid spreading CRS handling explanation in different places (#4694)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_projections/working_with_projections.rst

created. For data supported by OGR or GDAL, QGIS relies on the presence of a
recognized means of specifying the CRS. For instance, for the Shapefile format

> this is a file containing the Well-Known Text (:index:`WKT`)

Author: There's also ESRI WKT, see eg https://spatialreference.org/ref/epsg/27700/

> this is a file containing the Well-Known Text (:index:`WKT`)

Reviewer: When we write "WKT" the readers will expect the standard, as proposed by the OGC and adapted by ISO TC211 and SQL MM/Spatial. So we must either avoid WKT, or use "ESRI WKT".

     there's no prompt to select a CRS when a layer without CRS is loaded,
     defering CRS choice to a later time. Convenient when loading a lot of
     layers at once. Such layers will be identifiable in the :guilabel:`Layers`
     panel by the |indicatorNoCRS| icon next to them.
     They'll also be un-referenced, with coordinates from the layer treated as
     purely numerical, non-earth values, i.e. the same behavior as all layers
     get when :ref:`a project is set to have no CRS <project_crs>`.
   * |radioButtonOff| :guilabel:`Prompt for CRS`: it will prompt you to
     manually select the CRS. Selecting the correct choice is crucial,
     as a wrong choice will place your layer in the wrong position on the
     Earth's surface! Sometimes, accompanying metadata will describe the
     correct CRS for a layer, in other cases you will need to contact the
     original author of the data to determine the correct CRS to use.
   * |radioButtonOff| :guilabel:`Use project CRS`
   * |radioButtonOff| :guilabel:`Use default layer CRS`, as set in the
     :guilabel:`Default CRS for layers` combobox above.

.. tip::

>    To assign at once the same CRS to multiple layers that have no crs

Reviewer: ```suggestion
   To assign the same CRS to multiple layers that have no crs
```

Author: the idea is to do it in a row, which does not appear in the suggestion

Reviewer: Well, I think that my suggestion combined with your recipe below is clear - multiple layers will be assign the CRS, and then it is implicit that it is done in one go.  "at once" has a slightly different meaning - it is more like "immediately".

>    or have a wrong one:

Reviewer: ```suggestion
   or have a wrong one in one operation:
```

   #. Select the layers in the :guilabel:`Layers` panel

>    #. Press :kbd:`Ctrl+Shift+C`. You could also right-click over a layer

Reviewer: ```suggestion
   #. Press :kbd:`Ctrl+Shift+C`. You could also right-click over one of the selected layers
```

>       or go to :menuselection:`Layer --> Set CRS of layer(s)`

Author: ```suggestion
      selected layers or go to :menuselection:`Layer --> Set CRS of layer(s)`
```

   #. Find and select the right CRS to use

>    #. And press :guilabel:`OK`. The new CRS is now mentioned in the

Reviewer: ```suggestion
   #. And press :guilabel:`OK`. The new CRS is now described in the
```

Author: Not convinced: the crs is listed in the dialog, not really a description

Reviewer: "mentioned" is too vague in my opinion.

>       :guilabel:`Source` tab of the layer's properties dialog.

Reviewer: ```suggestion
      :guilabel:`Source` tab of the layers' properties dialog.
```

   Note that changing the CRS in this setting does not alter the underlying
   data source in any way, rather it just changes how QGIS interprets the raw

>    coordinates from the layer in the current QGIS project only.

Reviewer: ```suggestion
   coordinates from the layer in the current QGIS project.
```

.. index:: CRS, Proj, On-the-fly transformation
   single: CRS; Default CRS
.. _project_crs:

Project Coordinate Reference Systems
====================================

Every project in QGIS also has an associated Coordinate Reference System.
The project CRS determines how data is projected from its underlying raw
coordinates to the flat map rendered within your QGIS map canvas.

QGIS supports "on the fly" CRS transformation for both raster and vector data.
This means that regardless of the underlying CRS of particular map layers in
your project, they will always be automatically transformed into the common
CRS defined for your project. Behind the scenes, QGIS
transparently reprojects all layers contained within your project into the
project's CRS, so that they will all be rendered in the correct position with
respect to each other!

It is important to make an appropriate choice of CRS for your QGIS projects.
Choosing an inappropriate CRS can cause your maps to look distorted,
and poorly reflect the real-world relative sizes and positions of features.
Usually, while working in smaller geographic areas, there will be a number of

> standard CRSs used within a particular country or administrative area.

Reviewer: Should we mention the QGIS algorithm ("Find Projection", under QGIS Vector General in the processing toolbox) that lists appropriate CRSs for a specific area?

Author: we could

.. figure:: img/projectionDialog.png
   :align: center

   Project Properties Dialog

Available options are:

* |unchecked| :guilabel:`No projection (or unknown/non-Earth projection)`:
  Checking this setting will disable ALL projection handling within the QGIS
  project, causing all layers and map coordinates to be treated as simple 2D
  Cartesian coordinates, with no relation to positions on the Earth's surface.
  It can be used to guess a layer CRS (based on its raw coordinates or when
  using QGIS for non earth uses like role-playing game maps, building mapping
  or microscopic stuff. In this case:

  * No reprojection is done while rendering the layers: features are just drawn
    using their raw coordinates.
  * The ellipsoid is locked out and forced to ``None/Planimetric``.
  * The distance and area units, and the coordinate display are locked out and
    forced to "unknown units"; all measurements are done in unknown map units,
    and no conversion is possible.

* or an existing coordinate reference system that can be *geographic*, *projected*
  or *user-defined*. A preview of the CRS extent on earth is displayed to
  help you select the appropriate one.

>   Layers added to the project are on-the-fly translated to this CRS in order

Reviewer: ```suggestion
  Layers added to the project are "on-the-fly" translated to this CRS in order
```

Author: I wonder if we really need to quote this expression

Reviewer: Just trying to be consistent.

Author: my preference would then be to go the other way

Reviewer: Fine by me.

>   to overlay them regardless their original CRS. Use of units, ellipsoid setting

Reviewer: ```suggestion
  to overlay them regardless their original CRS. Use of units and ellipsoid setting
```
